build.gradle

```diff
@@ -17,15 +17,16 @@ buildscript {
   }
 }
 
+apply plugin: 'kotlin'
+apply plugin: 'java-gradle-plugin'
+apply plugin: 'com.jfrog.bintray'
+apply plugin: 'maven-publish'
```

Comment:
Is this one really required already having the Bintray one?

Reply:
It is if I want to publish locally